src/Jackett.Common/Definitions/hdbao.yml

```
@@ -0,0 +1,297 @@
+    # listing by date in both directions in the web UI of the exampale
+    # I copied. Where can I see the need for this in the example I
+    # copied so I can identify when it's needed for future
+    # definitions? Is it necessary for this definition?
+    date_added:
+      # time added
+      selector: td:nth-child(4):not(:has(span))
+      optional: true
+      filters:
+        - name: append
+          args: " +08:00" # CST
+        - name: dateparse
+          args: "yyyy-MM-ddHH:mm:ss zzz"
+    date:
+      text: "{{ if or .Result.date_elapsed .Result.date_added }}{{ or .Result.date_elapsed .Result.date_added }}{{ else }}now{{ end }}"
+    size:
+      selector: td:nth-child(5)
+    seeders:
+      selector: td:nth-child(6)
+    leechers:
+      selector: td:nth-child(7)
+    grabs:
+      selector: td:nth-child(8)
+    # TODO: There are no results for "2X", "2X 50%", and "30%" so I
+    # can't confirm their selectors but there are other examples in
+    # the example I copied of adding presumprive support for `imdbid`
+    # and `doubanid`, so I included them here.
+    #
+    # TODO: While testing, I could only find instances of "2X Free"
+    # and "ordinary" when I clicked through each of the top-level
+    # categories above, so those two are the only I could confirm by
+    # testing in the Prowlarr search UI. Is there a way I could have
+    # confirmed the rest?
+    downloadvolumefactor:
+      case:
+        img.pro_free: 0
+        img.pro_free2up: 0
+        img.pro_50pctdown: 0.5
+        img.pro_50pctdown2up: 0.5
+        img.pro_30pctdown: 0.3
+        "*": 1
+    uploadvolumefactor:
+      case:
+        img.pro_50pctdown2up: 2
+        img.pro_free2up: 2
+        img.pro_2up: 2
+        "*": 1
+    minimumseedtime:
+      # special 7 days (as seconds = 7 x 24 x 60 x 60)
+      # normal 3 days (as seconds = 3 x 24 x 60 x 60)
+      case:
+        # TODO: I found an instance of this in the example I copied by
+        # sorting by reverse date but that didn't yield an example for
+        # this indexer. See above regarding presumptive support. Am I
+        # correct to leave this in?
+        img[title="H&R"]: 604800
+        "*": 259200
+    description_verified:
+      case:
+        # TODO: I don't see these icons in this indexer's web UI but I
+        # do in the example I copied. Another case of presumptive
+        # support, correct?
+        span[title="通过"], span[title="通過"], span[title="Allowed"]: "Verified:"
+        span[title="未审"], span[title="未審"], span[title="Not reviewed"]: "Unverified:"
+        # TODO: Another case I can't find an instance of in the example I copide. How do I find an insance?
+        span[title="拒绝"], span[title="拒絕"], span[title="Denied"]: "Banned:"
+    description:
+      selector: td:nth-child(2)
+      remove: a, b, font, img, span
+      filters:
+        - name: prepend
+          args: "{{ .Result.description_verified }} "
```

```suggestion
    description:
      selector: td:nth-child(2)
      remove: a, b, font, img, span
```

It's the ticks/crosses/warning/question mark at the end of the titles (on some trackers it's at the start or has its own column). They don't exist here.

---

But from what I've observed, this is a NexusPHP feature, no? If yes, then this tracker may use it at some point and it doesn't seem very fragile to keep this support in. Why do some things get presumptive support and others not?

---

Oh, NM. It has to be removed from the definition when not in use so as not to pollute the row results. Sorry.